toolkits/brightdata/brightdata/tools/bright_data_tools.py

```diff
@@ -0,0 +1,275 @@
+import json
+import time
+from typing import Annotated, Dict, Optional
+
+import requests
+from arcade_tdk import ToolContext, tool
+
+from ..bright_data_client import BrightDataClient
+
+
+@tool(requires_secrets=["BRIGHTDATA_API_KEY", "BRIGHTDATA_ZONE"])
+def scrape_as_markdown(
+    context: ToolContext,
+    url: Annotated[str, "URL to scrape"],
+) -> Annotated[str, "Scraped webpage content as Markdown"]:
+    """
+    Scrape a webpage and return content in Markdown format using Bright Data.
+    
+    Examples:
+        scrape_as_markdown("https://example.com") -> "# Example Page\n\nContent..."
+        scrape_as_markdown("https://news.ycombinator.com") -> "# Hacker News\n..."
+    """
+    api_key = context.get_secret("BRIGHTDATA_API_KEY")
+    zone = context.get_secret("BRIGHTDATA_ZONE")
+    client = BrightDataClient.create_client(api_key=api_key, zone=zone)
+    
+    payload = {"url": url, "zone": zone, "format": "raw", "data_format": "markdown"}
+    return client.make_request(payload)
+
+
+@tool(requires_secrets=["BRIGHTDATA_API_KEY", "BRIGHTDATA_ZONE"])
+def get_screenshot(
```

Comment:
This tool is interesting because it only works if someone is using this tool locally and creating the file on their own machine.